riscv-rt/build.rs

```diff
@@ -100,6 +104,7 @@ fn main() {
 
         // expose the ISA extensions
         for ext in &extensions {
+            println!("cargo::rustc-check-cfg=cfg(riscv{ext})");
```

Comment:
This is not enough - it will only add the extensions of the current target, not all the extensions we may be checking for.

Reply:
Right, this lint check is new to me, and I thought it was for checking expected `cfg` output.

After re-reading the docs, it's clear now that it's meant to capture **all** expected `cfg` output.

I'll fix it in the next set of changes.